packages/framework/src/Resources/views/Admin/Content/Payment/qrCode.html.twig

```diff
@@ -0,0 +1,13 @@
+<div>
+    <p>
+        {% trans into orderLocale %}Payment instructions for bank transfer and QR payment{% endtrans %}
+    </p>
+    <p>
+        <img src="{{ qrCodeImageSrc }}" alt="{% trans into orderLocale %}QR code{% endtrans %}" class="qr-code" />
+    </p>
+    <p>
+        {% trans into orderLocale %}Total price{% endtrans %}: {{ totalPrice.priceWithVat|priceTextWithCurrencyByCurrencyIdAndLocale(order.currency.id, orderLocale)|hidePrice(order.customerUser) }}
+    </p>
```

Comment:
the `hidePrice` filter brings an interesting question - should we even display the qr code when the user does not have permissions to see the price? :smile: